src/XMLimport.cpp

```diff
@@ -824,27 +816,14 @@ void XMLimport::readHostPackage(Host* pHost)
     pHost->setUserDictionaryOptions(enableUserDictionary, useSharedDictionary);
     pHost->mShowInfo = (attributes().value("mShowInfo") == "yes");
     pHost->mAcceptServerGUI = (attributes().value("mAcceptServerGUI") == "yes");
-    if (attributes().hasAttribute(QLatin1String("mAcceptServerMedia"))) {
-        pHost->mAcceptServerMedia = (attributes().value("mAcceptServerMedia") == "yes");
-    }
+    pHost->mAcceptServerMedia = (attributes().value("mAcceptServerMedia") == "yes");
     pHost->mMapperUseAntiAlias = (attributes().value("mMapperUseAntiAlias") == "yes");
-    pHost->mMapperShowRoomBorders = attributes().value("mMapperShowRoomBorders") == "yes" || !attributes().hasAttribute("mMapperShowRoomBorders");
-    if (attributes().hasAttribute(QStringLiteral("mEditorAutoComplete"))) {
-        pHost->mEditorAutoComplete = (attributes().value(QStringLiteral("mEditorAutoComplete")) == "yes");
-    }
-    if (attributes().hasAttribute(QLatin1String("mEditorTheme"))) {
-        pHost->mEditorTheme = attributes().value(QLatin1String("mEditorTheme")).toString();
-    }
-    if (attributes().hasAttribute(QLatin1String("mEditorThemeFile"))) {
-        pHost->mEditorThemeFile = attributes().value(QLatin1String("mEditorThemeFile")).toString();
-    }
-    if (attributes().hasAttribute(QLatin1String("mThemePreviewItemID"))) {
-        pHost->mThemePreviewItemID = attributes().value(QLatin1String("mThemePreviewItemID")).toInt();
-    }
-    if (attributes().hasAttribute(QLatin1String("mThemePreviewType"))) {
-        pHost->mThemePreviewType = attributes().value(QLatin1String("mThemePreviewType")).toString();
-    }
-
+    pHost->mMapperShowRoomBorders = readDefaultTrueBool("mMapperShowRoomBorders");
+    pHost->mEditorAutoComplete = (attributes().value(QStringLiteral("mEditorAutoComplete")) == "yes");
```

Comment:
TBH All these texts (even the `yes`) need wrapping in `QStringLiteral(...)` - though since there are multiples of that particular `QString` through out this file it would be better to have a `const QString YES{QStringLiteral("yes"}};` defined at the top of the file so that there is only one instance stored in the read-only part of the object code for this class.

Reply:
It's good to keep code simpler by having less indirection

Reply:
Added const and wrapped most of attributes value calls with `QStringLiteral`